Description: Remove unneeded Python includes
 Inluding <compile.h> and <eval.h> has not been necessary since Python 2.4,
 since they are included via <Python.h>.  Morever, <eval.h> is being removed in
 Python 3.11.  So remove these includes.
Author: Peter Eisentraut <peter@eisentraut.org>
Origin: upstream, https://github.com/postgres/postgres/commit/0d335cbd5c63009576280ff502b31a84e353a86b
Forwarded: no
Last-Update: 2023-02-04

--- a/src/pl/plpython/plpython.h
+++ b/src/pl/plpython/plpython.h
@@ -124,9 +124,6 @@
 #undef TEXTDOMAIN
 #define TEXTDOMAIN PG_TEXTDOMAIN("plpython")
 
-#include <compile.h>
-#include <eval.h>
-
 /* put back our *printf macros ... this must match src/include/port.h */
 #ifdef vsnprintf
 #undef vsnprintf
